### DISCUSSION TITLE
Document says: You cannot push a list of values - only a single value. But, It works?

### DISCUSSION THREAD

**Discussion by aliramazon** [prisma/prisma#21464] [Q&A] [answered]
### Question

Hi,

In Prisma documention: You cannot push a list of values - only a single value. But, in my case it did work.

```js
await prisma.post.update({where: {id}, data: {images: {push: [img1, img2]}})
```
 did work

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @aliramazon 👋 

Thank you for raising this question.

I can confirm that using `push` allows for adding a list of values. I created a simple model as shown below:
```prisma
model Post {
  id      Int      @id @default(autoincrement())
  title   String
  content String
  tags    String[]    
}
```
it generates a type of 
```ts
export type PostUpdatetagsInput = {
    set?: string[]
    push?: string | string[]
  }
```
The `push` method can either be used to add a single value or a list of values.
I've created a docs [issue](https://github.com/prisma/docs/pull/5369) to update this. 

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏
